aggregate_spatial_window.json

```diff
@@ -0,0 +1,95 @@
+{
+  "id": "aggregate_spatial_window",
+  "summary": "Zonal statistics for rectangular windows",
+  "description": "Aggregates statistics over the spatial dimensions. This process passes a list of values to the reducer.\n\nThe results are computed dividing the input X/Y grid into x_size/y_size non-overlapping windows and applying the reducer process to each window.\nIf X and Y are not multiple of x_size/y_size NaNs padding before the computation is applied.",
+  "categories": [
+    "cubes",
+    "aggregate & resample"
+  ],
+  "parameters": [
+    {
+      "name": "data",
+      "description": "A raster data cube.",
+      "schema": {
+        "type": "object",
+        "subtype": "raster-cube"
+      }
+    },
+    {
+      "name": "reducer",
+      "description": "A reducer to be applied on all values of each geometry. A reducer is a single process such as ``mean()`` or a set of processes, which computes a single value for a list of values, see the category 'reducer' for such processes.",
+      "schema": {
+        "type": "object",
+        "subtype": "process-graph",
+        "parameters": [
+          {
+            "name": "data",
+            "description": "An array with elements of any type.",
+            "schema": {
+              "type": "array",
+              "items": {
+                "description": "Any data type."
+              }
+            }
+          },
+          {
+            "name": "context",
+            "description": "Additional data passed by the user.",
+            "schema": {
+              "description": "Any data type."
+            },
+            "optional": true,
+            "default": null
+          }
+        ]
+      }
+    },
+    {
+      "name": "x_size",
```

Comment:
In `apply_neighborhood` we use `size = [2,5]` instead of x_size = 2 / y_size = 5. Should we align?

Reply:
yes, we can use the same as apply_neighborhood.

Reply:
Yeah, maybe we should just take over the definition from https://github.com/Open-EO/openeo-processes/blob/master/apply_neighborhood.json#L44-L91

It has its own subtype `chunk-size` and thus I envision good client support. We can remove the duration of course and may want to remove the null behavior (all values).